src/TideOperation.php

```diff
@@ -17,4 +36,11 @@ public static function enableStandaloneMedia() {
     $config->save();
   }
 
+  /**
+   * Enables entity_usage module.
+   */
+  public function enableEntityUsage() {
+    $this->tideMediaInstallModule('entity_usage');
```

Comment:
Can this be static like the other functions?

Reply:
I don't know why I did that, can't remember. But fixed now